### PR TITLE
Optimize Agile page some more

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
Fixes for reducing bounce rates

- [x] Change order of sections (What should be before Why, clients = proof and can get a better position etc.) 
- [x] Update hero copy (more specific)
- [x] Update form (the options was a bad idea)

### CODE REVIEW

File: source/localizable/capp-agile/_capp-agile-clients.erb [hunk 1]
```diff
@@ -1,45 +1,15 @@
...
-    </a>
+    <%= image_tag "clients/doppio.png", alt: "Doppio Espresso", title: "Doppio Espresso" %>
+    <%= image_tag "clients/radboudumc.png", alt: "Radboud Universitair Medisch Centrum", title: "Radboud Universitair Medisch Centrum" %>
+    <%= image_tag "clients/albertschweitzerziekenhuis.png", alt: "Albert Schweitzer Ziekenhuis", title: "Albert Schweitzer Ziekenhuis" %>
+    <%= image_tag "clients/kwintes.png", alt: "Kwintes", title: "Kwintes" %>
+    <%= image_tag "clients/nhl.png", alt: "NHL Hogeschool", title: "NHL Hogeschool" %>
+    <%= image_tag "clients/heliomare.png", alt: "Heliomare", title: "Heliomare" %>
+    <%= image_tag "clients/rivierenland.png", alt: "Ziekenhuis Rivierenland", title: "Ziekenhuis Rivierenland" %>
+    <%= image_tag "clients/umcg.png", alt: "Universitair Medisch Centrum Groningen", title: "Universitair Medisch Centrum Groningen" %>
+    <%= image_tag "clients/dpa-benkis.png", alt: "DPA Benkis", title: "DPA Benkis" %>
```
Comment (fatboypunk) [inline source/localizable/capp-agile/_capp-agile-clients.erb]: we don't link to them anymore?

**Comment by RonaldDijkstra** [comment]
Nvm

File: source/localizable/capp-agile/_capp-agile-clients.erb (hunk 1 above)
Comment (RonaldDijkstra) [inline source/localizable/capp-agile/_capp-agile-clients.erb]: Want users to stay..

**Comment by RonaldDijkstra** [comment]
Only thing left for this PR is the HERO copy.

**Comment by RonaldDijkstra** [comment]
Let's try it like this first. 

Goals: reduce bounce rate and some more demo requests, because less clutter. And then I'll work on a V2 of this page, which should breath **new awesome product**.

**Pull request merged**